pkg/resource/resource.go

```diff
@@ -67,10 +67,12 @@ var refactoredResources = []string{
 	"aws_sns_topic",
 	"aws_sns_topic_policy",
 	"aws_sns_topic_subscription",
-	// "aws_sqs_queue",
-	// "aws_sqs_queue_policy",
-	// "aws_subnet",
-	// "aws_vpc",
+	"aws_sqs_queue",
+	"aws_sqs_queue_policy",
+	"aws_subnet",
+	"aws_subnet",
+	"aws_default_subnet",
```

Comment:
Please remove one `aws_subnet` and there is already the `aws_default_subnet` line 28